lock/lock.py

```diff
@@ -359,43 +347,39 @@ async def unviewlock(
         `[p]unviewlock #hidden-channel true`
         `[p]unviewlock 746284923572835 @boosters`
         """
-        try:
-            await ctx.trigger_typing()
```

Comment:
Please use `await ctx.typing()` and retain the try/except block rather than using a context manager.